packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -114,14 +124,9 @@ const ProgressIndicatorItem = ({
 	const listItemLinkTextSelector = '> span:last-of-type > span:first-of-type';
 
 	if (isActive) {
-		const activeStatusWhiteList: Array<ProgressIndicatorItemStatus> = [
-			'blocked',
-			'doing',
-			'started',
-		];
-		if (!activeStatusWhiteList.includes(status)) {
+		if (!activeProgressStatusAllowedList.includes(status)) {
```

Comment:
Maybe make this a single `if` statement?

Reply:
fixed